internal/storage/block_storage.go

```diff
@@ -686,77 +687,6 @@ func (b *BlockStorage) BootstrapBalances(
 	return nil
 }
 
-// BalanceChanges returns all balance changes for
```

Comment:
The biggest piece of this PR is migrating to the `parser` package's `BalanceChanges`.